src/tsserver/server.ts

```diff
@@ -57,6 +58,7 @@ namespace ts.server {
         console.warn = (...args) => logger.msg(args.length === 1 ? args[0] : args.join(", "), Msg.Err);
         console.error = (...args) => logger.msg(args.length === 1 ? args[0] : args.join(", "), Msg.Err);
 
+        const fshost = vfs ? VirtualFS.createVirtualServerHost([]) : sys as ServerHost;

```

Comment:
Wonder if case sensitivity and new line should be passed in from `sys`

Reply:
excecuting file path ? (in directly controls the lib location)

Reply:
still need to test executing file path -- I'm using `""` until then